blog/2024-08-02/index.md

```diff
@@ -0,0 +1,41 @@
+---
+slug: 2-billion-downloads-2024
+title: 2 billion downloads, 4.7 million active users, 1k verified apps, and more stats from 2024 so far
+authors: [cassidyjames]
+tags: [milestone, stats]
+image: ./milestones.png
+---
+
+It's been a busy year, and our platform and developer community-building efforts are paying off. Let's take a look at what we've been up to over the last six months, and measure its effect.
+
+<!-- truncate -->
+
+At the beginning of the year we announced an incredible milestone for Flathub: over one million active users. But we didn't slow down; in February we attended FOSDEM, then announced improved app moderation and app brand colors in preparation for some new features. Those efforts ultimately launched with the new homepage and featured app banners in April.
+
+We're back with some new milestones thanks to the continued growth of Flathub as an app store and the incredible work of both our largely volunteer team and our growing app developer community:
+
+## 70% of the most popular apps are verified
+
+Over [1,000 apps](https://flathub.org/apps/collection/verified/) have been verified by their developers on Flathub, including 70% of the [top 30 most popular apps](https://flathub.org/apps/collection/popular/1). Developers of verified apps are ultimately in charge of their own app listings, and their updates are delivered directly to Flathub users while passing our automated testing and human review of things like permission changes.
+
+## 100+ apps meet stricter quality guidelines
+
+Over 100 apps are now passing our quality guidelines that include checks like icon contrast on both light and dark backgrounds, quality screenshots, and consistent app naming and descriptions so users get a better experience browsing Flathub. We're proud of the apps that have gone above and beyond here, and we expect to see even more apps opt in over time.
```

Comment:
I'd maybe mention why 100 is a lot in this context. It means that between late February and July, over 100 apps went out of their way to make significant changes to their metadata to get ready for the new guidelines.

Reply:
```suggestion
Over 100 apps are now passing our quality guidelines that include checks like icon contrast on both light and dark backgrounds, quality screenshots, and consistent app naming and descriptions so users get a better experience browsing Flathub. These guidelines are what enable us to display visually appealing and consistent banners on the new home page, for example.

This means that between late February and July, the developers of over 100 apps went out of their way to improve—and sometimes make significant changes to—their apps' metadata to get ready for these new guidelines and features on Flathub. We're proud of these developers who have gone above and beyond, and we look forward to even more apps opting in over time.
```